packages/react-components/src/messages/__tests__/Welcome.test.tsx

```diff
@@ -14,5 +14,15 @@ it('renders the welcome template with name and link', () => {
   );
 
   const heading = getByRole('heading');
+  const cta = getByRole('link', {
+    name: /activate account/i,
+  }) as HTMLAnchorElement;
```

Comment:
To make this case safe, the `selector: 'a'` option to getByRole should also be used

Reply:
Can you check the updated version?